tests/unit/certGen.unitTest.ts

```diff
@@ -8,14 +8,24 @@ import {S3BucketMockService} from "../models/S3BucketMockService";
 import {LambdaMockService} from "../models/LambdaMockService";
 import {CertificateUploadService} from "../../src/services/CertificateUploadService";
 import {ManagedUpload} from "aws-sdk/clients/s3";
+import {certGen} from "../../src/functions/certGen";
+import mockContext from "aws-lambda-mock-context";
+import sinon from "sinon";
+// tslint:disable
+const queueEventPass = require("../resources/queue-event-pass.json");
+const queueEventFail = require("../resources/queue-event-fail.json");
+const queueEvent = require("../resources/queue-event");
+// tslint:enable
+const ctx = mockContext();
+const sandbox = sinon.createSandbox();
 
 describe("cert-gen", () => {
     context("CertificateGenerationService", () => {
         const certificateGenerationService: CertificateGenerationService = Injector.resolve<CertificateGenerationService>(CertificateGenerationService, [S3BucketMockService, LambdaMockService]);
         LambdaMockService.populateFunctions();
 
         context("when a passing test result is read from the queue", () => {
-            const event: any = JSON.parse(fs.readFileSync(path.resolve(__dirname, "../resources/queue-event-pass.json"), "utf8"));
+            const event: any = {...queueEventPass};
```

Comment:
can we possibly specify inferred type for event, if quick and easy or add tslint rules for now and noted to be fixed later